mathics/builtin/arithfns/basic.py

```diff
@@ -528,15 +538,15 @@ class Power(BinaryOperator, _MPMathFunction):
     # Remember to up sympy doc link when this is corrected
     sympy_name = "Pow"
 
+    def eval_exp(self, x, evaluation):
+        "Power[E, x]"
+        return eval_exponential(x)
```

Comment:
Shouldn't this be eval_Exponential() ? (And same for eval_power_*.

Reply:
yes, it should. However, what I did here was just to rebase this PR, in order to split this into smaller parts. As I will split the small pieces, I plan to fix these name conventions.